lib/AddFiles.pm

```diff
@@ -998,4 +996,64 @@ sub replace_config_var
   return "<$name>";
 }
 
+
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# Set up /proc and /dev/fd in directory dir if they are missing as a number
+# of tools rely on these (bsc#1160594, bsc#1176972).
```

Comment:
when doing it why not proc mount also "/sys" I think some tools also use it. Otherwise change looks reasonable.

Reply:
The main reason here is the secure access to file descriptors via `/proc/self`. `/sys` exposes too much of the host system and it would kind of contradict the use of a chroot environment.

Note that using `/proc` is also not without problems as it already exposes quite a lot and processes might influence the host system in unwanted ways.

It's a compromise already and I would not want to stretch it further.

Reply:
ok